app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,13 @@
 <resources>
     <string name="app_name">take-my-money</string>
+    <string name="bitcoin">Bitcoin</string>
+    <string name="Btc">btc</string>
```

Comment:
Strings name should be all lower_case
Nomes de string deveria ser todas em minúsculas